js/apps/filter-page/controllers/filterPageCtrl.js

```diff
@@ -114,27 +125,18 @@
 			}
 		};
 
-		const showFilters = (collapseEvent) => {
+		const toggleIcon = (collapseEvent) => {
 			const $collapsible = angular.element(collapseEvent.currentTarget);
-			const $collapseControl = $collapsible.siblings('.collapse-control');
-
-			$collapseControl.html('<i class="fa fa-minus"></i> Hide Filters');
-		};
-
-		const hideFilters = (collapseEvent) => {
-			const $collapsible = angular.element(collapseEvent.currentTarget);
-			const $collapseControl = $collapsible.siblings('.collapse-control');
-
-			$collapseControl.html('<i class="fa fa-plus"></i> Show Filters');
+			const $collapseIcon = $collapsible.closest('.expando-wrapper').find('i');
+			$collapseIcon.toggleClass('fa-plus-square').toggleClass('fa-minus-square');
```

Comment:
ngClass?

Reply:
I'm not seeing a way to do this that won't require a big rewrite. I'm open to ideas.

Reply:
This was bugging me, you could use this [library](https://github.com/angular-ui/bootstrap). Here is a [plunker](https://plnkr.co/edit/ovD68JU58wMASm6psdhp?p=preview) with similar functionality to what you are looking for (I think).